hedera-mirror-rest/__tests__/specs/network-fees-03-timestamp-upper-bound.spec.json

```
@@ -0,0 +1,65 @@
+{
+  "description": "Network fees with upper bound timestamp",
```

nit: urls also include eq case. So you can update the description or split the files. I say update the description since the response is the same as written
```suggestion
  "description": "Network fees with inner and upper bound timestamp",
```